node-installer/script/installer.sh

```diff
@@ -66,7 +66,7 @@ cp /assets/containerd-shim-spin-v2 $NODE_ROOT$KWASM_DIR/bin/
 # A bug in containerd makes BinaryName not work with shim not in PATH, so this statically links the kwasm installation to path
 # https://github.com/containerd/containerd/issues/11480
 mkdir -p $NODE_ROOT/usr/local/bin/
-ln -s $KWASM_DIR/bin/containerd-shim-spin-v2 $NODE_ROOT/usr/local/bin/containerd-shim-spin
```

Comment:
How did this ever work? Or is this change not released yet?

Reply:
Not released yet but I am not sure how the CI was not able to detect it

Reply:
Oh wait, the CI was skipped for this change - I don't know why though: https://github.com/spinframework/containerd-shim-spin/actions/runs/14123812940. But this explains why this error went undetected to the main tree